extensions/git/src/git.ts

```diff
@@ -338,7 +338,7 @@ export class Git {
 	}
 
 	async clone(url: string, parentPath: string, cancellationToken?: CancellationToken): Promise<string> {
-		let baseFolderName = decodeURI(url).replace(/^.*\//, '').replace(/\.git$/, '') || 'repository';
+		let baseFolderName = decodeURI(url).replace(/\/$/, '').replace(/^.*\//, '').replace(/\.git$/, '') || 'repository';
```

Comment:
Is there ever a situation where there could be multiple slashes at the end of the URL? I can't think of any but I also can't think of a situation where removing multiple would be a bad idea. In other words, maybe consider changing `/\/$/` to `/\/*$/`. Just a thought though.